src/views/Auth/components/DataManager.vue

```diff
@@ -70,17 +71,19 @@
       :title="isEdit ? $t('Base.edit') : $t('Base.add')"
       width="480px"
       v-model="dialogVisible"
+      @open="handleDialogOpen"
     >
       <el-form ref="recordForm" :model="record" :rules="getRules()" label-position="top">
         <el-form-item prop="user_id" :label="getFiledLabel(field)">
-          <el-input v-model="record.user_id" :disabled="isEdit"></el-input>
+          <el-input size="small" v-model="record.user_id" :disabled="isEdit"></el-input>
```

Comment:
Can this property be set on the form component?

Reply:
Good suggestion

Reply:
But the size of the UI components, we need to discuss and keep consistent during the redesign

Reply:
😲